docs/stable/store/sllm-store-cli.md

```diff
@@ -0,0 +1,194 @@
+---
+sidebar_position: 2
+---
+
+# ServerlessLLM Store CLI
+
+ServerlessLLM Store's CLI allows the use `sllm-store`'s functionalities within a terminal window. It has the functions:
+- `save`: Convert a HuggingFace model into a loading-optimized format and save it to a local path.
+- `load`: Load a model into given GPUs.
+
+
+## Requirements
+- OS: Ubuntu 20.04
+- Python: 3.10
+- GPU: compute capability 7.0 or higher
+
+## Installations
+
+### Create a virtual environment
+```bash
+conda create -n sllm-store python=3.10 -y
+conda activate sllm-store
+```
+
+### Install C++ Runtime Library (required for compiling and running CUDA/C++ extensions)
+``` bash
+conda install -c conda-forge libstdcxx-ng=12 -y
+```
+
+### Install with pip
+```bash
+pip install serverless-llm-store
+```
+
+### Install from source
```

Comment:
This is the API documentation. Let's keep it minimal with just a few examples and basic installation instructions using `pip`.